zScanner/Services/Network/Medicalc/Requests/LoginRequest.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  LoginRequest.swift
+//  zScanner
+//
+//  Created by Ales Teska on 29.2.20.
+//  Copyright © 2020 Institut klinické a experimentální medicíny. All rights reserved.
+//
+
+import Foundation
+
+struct LoginRequest: Request {
+    typealias DataType = EmptyResponse
+    
+    var endpoint: Endpoint = SeaCatPKIEndpoint.login
+    var method: HTTPMethod = .post
+    var parameters: Parameters? = nil
+    var headers: HTTPHeaders = [:]
+    
+    init(username: String, password: String) {
+        //This has to go in the body!!! of the POST request
```

Comment:
Conform `LoginRequest` to the protocol `ParametersJsonEncoded `

Reply:
A sample code is welcomed ;-)